src/amp-shadow-babel.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Well, I know it is there, but it doesn't need to be :)

You can leave this. I should get rid of them.